src/qmckl_context.org

```diff
@@ -359,6 +363,10 @@ COEFFICIENT = [ 0.006068, 0.045308, 0.202822, 0.503903, 0.383421,
      #+END_EXAMPLE
 
 **** ~qmckl_context_update_ao_basis~
+     :PROPERTIES:
+     :Type:     qmckl_exit_code
+     :Name:     qmckl_context_update_ao_basis
+     :END:
 
      Updates the data describing the AO basis set into the context.
 
```

Comment:
We can remove the old table below

Reply:
Yes, I'll do the cleaning before we can merge this to the main branch. I'll also run the tests to check nothing is breaking.